### PR TITLE
Added back options to generate page, fixes #86

### LINKED ISSUE #86: Generate page: Show a back button/link
Right now, users are asked to use their back button. Have a button or link on the page that provides and alternate way to do this.

### PR DESCRIPTION
**Pull request by plusjeff**
Fixes #86

image

Highlights
- Go back text in initial description
- Go back text at botton
- Right alignment of upload button

### CODE REVIEW

**Comment by plusjeff** [comment]
@KHarshawat : Ready for your review

File: app/views/web/generate.html.erb [hunk 1]
```diff
@@ -6,8 +6,7 @@
         Review the completed Form 8 below and verify all fields are accurate.
         If the form looks correct, click the <strong>Upload and Certify</strong>
         button and the Form 8 will automatically be uploaded to VBMS. If there
-        are any mistakes, use your browser's back button to go back and correct
-        them.
+        are any mistakes, <a onlick="window.history.back();">go back the errors</a>.
```
Comment (alex) [inline app/views/web/generate.html.erb]: Does this need a `return false;`

**Comment by plusjeff** [comment]
Wait - typos. Fixing. :smile:

**Comment by plusjeff** [comment]
@alex Thanks! Fixed the go back links.

@KHarshawat Updated screenshot without typos:

image

**Pull request merged**